pkg/filters/filter.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package filters
+
+import (
+	cuckoo "github.com/seiflotfy/cuckoofilter"
+)
+
+// Filter is an interface for filters
+type Filter interface {
+	Contains(pieceID []byte) bool
+	Add(pieceID []byte)
+	Encode() []byte
+}
+
+// CuckooFilter is a cuckoo filter
+type CuckooFilter struct {
```

Comment:
I'll move this implementation to a different file, for example to the one currently named _bloom_filters.go_ and then rename it to something like _external_filters.go_ or _third_party_filters.go_.